components/admin_console/admin_definition.jsx

```diff
@@ -521,7 +521,7 @@ export default {
                         },
                         {
                             type: Constants.SettingsTypes.TYPE_BOOL,
-                            key: 'EnableDiagnostics',
+                            key: 'LogSettings.EnableDiagnostics',
```

Comment:
Im not seeing the settings for the json format for the logs, are they going to be included?

Reply:
Only if whoever added that setting wants to add it. This PR isn't adding any new settings. It's just changing how the existing ones are defined so I can add more in my other PR